schedule/create/main_dacx.go

```diff
@@ -51,5 +51,9 @@ id: how-to-create-a-schedule-in-go
 title: How to create a Schedule in Go
 label: Create Schedule
 description: To create a Schedule in Go, use `Create()` on the Client.
+tags: 
+	- go-sdk
+	- code-sample
+	- schedules
```

Comment:
```suggestion
	- schedule
```
singular here and the other ones -